File: java-service/src/main/java/models/Object_.java
package models;

public class Object_ {
    private boolean isInterface;
    private boolean isAnnotation;
    private String simpleName;
    private String fullName;
    private String filePath;
    private String serviceName;


    private String content;

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getSimpleName() {
        return simpleName;
    }

    public void setSimpleName(String simpleName) {
        this.simpleName = simpleName;
    }

    public String getFullName() {
        return fullName;
    }

    public void setFullName(String fullName) {
        this.fullName = fullName;
    }

    public String getFilePath() {
        return filePath;
    }

    public void setFilePath(String filePath) {
        this.filePath = filePath;
    }

    public String getServiceName() {
        return serviceName;
    }

    public void setServiceName(String serviceName) {
        this.serviceName = serviceName;
    }

    public boolean isInterface() {
        return isInterface;
    }

    public void setInterface(boolean anInterface) {
        isInterface = anInterface;
    }

    public boolean isAnnotation() {
        return isAnnotation;
    }

    public void setAnnotation(boolean annotation) {
        isAnnotation = annotation;
    }


}

File: csharp-service/MicroAnalyzer/models/Object_.cs
namespace MicroAnalyzer.models;

public class Object_
{
    public bool isInterface { get; set; }
    public bool isAnnotation { get; set; }
    public string simpleName { get; set; }
    public string fullName { get; set; }
    public string filePath { get; set; }
    public string serviceName { get; set; }
    public string content { get; set; }
    
    public Object_()
    {
    }
    
    public Object_(string simpleName, string fullName, string filePath, string serviceName, bool isInterface = false, 
        bool isAnnotation = false)
    {
        this.simpleName = simpleName;
        this.fullName = fullName;
        this.filePath = filePath;
        this.serviceName = serviceName;
        this.isInterface = isInterface;
        this.isAnnotation = isAnnotation;
    }
}